study-builder/studies/osteo/study.conf

```diff
@@ -2424,7 +2424,107 @@
     },
 
     # reconsent events
-
+    {
+      "trigger": {
+        "type": "REACHED_AOM"
+      },
+      "action": {
+        "type": "ACTIVITY_INSTANCE_CREATION",
+        "activityCode": "CONSENT"
+      },
+      "dispatchToHousekeeping": false,
+      "order": 1
+    },
+    {
+      "trigger": {
+        "type": "REACHED_AOM"
+      },
+      "action": {
+        "type": "MARK_ACTIVITIES_READ_ONLY",
+        "activityCodes": ["PARENTAL_CONSENT", "CONSENT_ASSENT", "RELEASE_MINOR", "ABOUTCHILD", "CHILD_CONTACT"]
+      },
+      "dispatchToHousekeeping": false,
+      "order": 1
+    },
+    # reconsent next steps email
+    {
+      "trigger": {
+        "type": "REACHED_AOM"
+      },
+      "action": {
+        "type": "INVITATION_EMAIL",
+        "emailTemplate": ${emails.reconsentNextSteps},
+        "language": "en",
+        "pdfAttachments": []
+      },
+      "dispatchToHousekeeping": true,
+      "order": 1
+    },
+    # reconsent/next steps reminder emails
+    {
+      "trigger": {
+        "type": "REACHED_AOM"
+      },
+      "action": {
+        "type": "INVITATION_EMAIL",
+        "emailTemplate": ${emails.reconsentNextSteps1wkReminder},
+        "language": "en",
+        "pdfAttachments": []
+      },
+      "cancelExpr": """user.studies["CMI-OSTEO"].forms["CONSENT"].isStatus("COMPLETE")""",
+      "delaySeconds": 604800, # one week in seconds
+      "dispatchToHousekeeping": true,
+      "order": 1
+    },
+    {
+      "trigger": {
+        "type": "REACHED_AOM"
+      },
+      "action": {
+        "type": "INVITATION_EMAIL",
+        "emailTemplate": ${emails.reconsentNextSteps2wkReminder},
+        "language": "en",
+        "pdfAttachments": []
+      },
+      "cancelExpr": """user.studies["CMI-OSTEO"].forms["CONSENT"].isStatus("COMPLETE")""",
+      "delaySeconds": 1209600, # two weeks in seconds
+      "dispatchToHousekeeping": true,
+      "order": 1
+    },
+    {
+      "trigger": {
+        "type": "REACHED_AOM"
+      },
+      "action": {
+        "type": "INVITATION_EMAIL",
+        "emailTemplate": ${emails.reconsentNextSteps4wkReminder},
+        "language": "en",
+        "pdfAttachments": []
+      },
+      "cancelExpr": """user.studies["CMI-OSTEO"].forms["CONSENT"].isStatus("COMPLETE")""",
+      "delaySeconds": 2419200, # four weeks in seconds
+      "dispatchToHousekeeping": true,
+      "order": 1
+    },
+    #reconsent complete thank-you
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "CONSENT",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "SENDGRID_EMAIL",
+        "emailTemplate": ${emails.reconsentThankYou},
+        "linkedActivityCode": "CONSENT",
```

Comment:
The reconsent thank-you email doesn't have a button that links to an activity (it links to dashboard), so we don't need to set `linkedActivityCode`.